### PR TITLE
[Table sanity check] time

### LINKED ISSUE #5063: Create tests for the table `time`
## Create tests for the table `time`

  - Create header file for the table implementation, if one is not exists.
  - In test, query the table and check if retrieved columns (name and types) match the columns from table spec.
  - If there is any guarantee to number of rows (e.g. only 1 record in every query result, more than 3 records or something else) check it.
  - Test the implementation details of the table, if it possible.

Table spec: `specs/utility/time.table`

Source files:

  - `osquery/tables/utility/time.cpp`

Table generating function: `genTime()`

### PR DESCRIPTION
**Pull request by fmanco**
Fixes #5063

### CODE REVIEW

**Comment by fmanco** [comment]
Jenkins test linux

File: osquery/tests/integration/tables/time.cpp [hunk 1]
```diff
@@ -16,36 +15,30 @@
...
+TEST_F(Time, test_sanity) {
+  QueryData data = execute_query("select * from time");
+
+  ASSERT_EQ(data.size(), 1ul);
+
+  ValidatatioMap row_map = {
+      {"weekday", NonEmptyString},
+      {"year", IntType},
+      {"month", IntType},
+      {"day", IntType},
```
Comment (mkareta) [inline osquery/tests/integration/tables/time.cpp]: Please use IntMinMax check for numeric values

File: osquery/tests/integration/tables/time.cpp [hunk 2]
```diff
@@ -16,36 +15,30 @@
...
+      {"hour", IntType},
+      {"minutes", IntType},
+      {"seconds", IntType},
+      {"timezone", NonEmptyString},
+      {"local_time", IntType},
+      {"local_timezone", NonEmptyString},
+      {"unix_time", IntType},
+      {"timestamp", NonEmptyString},
```
Comment (mkareta) [inline osquery/tests/integration/tables/time.cpp]: is timestamp a number?

Comment (fmanco) [reply osquery/tests/integration/tables/time.cpp]: No, timestamp is something like `Wed Aug 29 16:58:23 2018 UTC`.

**Pull request merged**